doc/data/messages/f/fixme/details.rst

```diff
@@ -1 +0,0 @@
-You can help us make the doc better `by contributing <https://github.com/PyCQA/pylint/issues/5953>`_ !
```

Comment:
It's to get crafty with regex to force the presence of a ticket number, for example see https://github.com/PyCQA/pylint/issues/2874#issuecomment-1177524718

Reply:
Oh not. That sounds bad... Might be able to recover some from your fork?

Reply:
Yeah some branches I pushed semi regularly, everything experimental or unfinished though... (especially a massive invalid-name refactor for name that were too small...) Well at least I won't have to clean up the git repo anymore and I get to start from a clean slate 🥲 I had a disk space problem for some time, and after months of just deleting the cache from time to time I think I finally had something important that did not get written correctly (auth app crashed) 😅